content/en/integrations/guide/source-code-integration.md

```diff
@@ -415,10 +415,10 @@ Setting up the GitHub integration also allows you to see inline code snippets in
 {{% tab "GitLab" %}}
 
 <div class="alert alert-warning">
-Repositories from self-managed GitLab instances are not supported out-of-the-box by the source code integration. To enable this feature, <a href="/help">contact Support</a>.
+Repositories from GitLab instances are supported in closed preview. <a href="https://www.datadoghq.com/product-preview/gitlab-source-code-integration/">Join the preview</a>.
```

Comment:
```suggestion
Repositories from GitLab instances are supported in closed Preview. <a href="https://www.datadoghq.com/product-preview/gitlab-source-code-integration/">Join the Preview</a>.
```

Reply:
Sorry - sent along my review and then remembered that we usually capitalize `Preview` 🤦🏻‍♀️